### DISCUSSION TITLE
Html sanitizer converts #hex color code to rgba

### DISCUSSION THREAD

**Discussion by amishra1911** [mganss/HtmlSanitizer#266] [General]
Hello Team,

I am using the sanitize method of HtmlSanitizer to sanitize the html code.
I noticed the #hex code present in style tag got converted to rgba.
Is there any way to prevent this conversion?
e.g-
<td bgcolor="#0079c1" style="border-top: #717171 1px solid......
got conevrted to
<td bgcolor="#0079c1" style="border: 1px solid rgba(113, 113, 113, 1)

**Comment by mganss** [comment]
Unfortunately, there isn't currently. It's the result of AngleSharp's CSS parser which behaves in the same way as browsers.

**Reply by jannikbuschke** [reply]
It is indicated that this would be possible by using a custom formatter: https://github.com/AngleSharp/AngleSharp/issues/132#issuecomment-734950426

It seems you already use a custom formatter, maybe it would be a rather easy fix. Are you open for PRs ?

**Reply by mganss** [reply]
Sure, but though we do have `StyleFormatter` now I believe this particular issue is hard to fix at that level. The easiest fix/workaround in the current version is the one described in #330, assuming you can live with all color values output as hex.

**Comment by lukos** [comment]
It is a pain, if you sanitize for an html email, it creates rgba which various email clients don't support! I'll have to look at Anglesharp then.

**Comment by mganss** [comment]
See #330 for a workaround/fix.
